doc/i18n.md

```diff
@@ -100,6 +100,8 @@ On Mudita Pure phone we use:
 - default font (`gt_pressura` font family)
 - default fallback font (DeJavu Sans Bold size 27 **Emojis only**)
 
+On Mudita Harmony we use only DeJavu Sans font family
```

Comment:
1. There is no GT pressura on harmony? :o 
2.  i18n is just a product-agnostic library - why add it here? add it to the product specification if required ( Pure Phone features are linked from main readme )